src/accountlist/accountlistReducers.js

```diff
@@ -0,0 +1,57 @@
+import { fromJS } from 'immutable';
+
+import {
+  REALM_ADD,
+  LOGIN_SUCCESS,
+  LOGOUT,
+  ACCOUNT_REMOVE,
+} from '../account/accountActions';
+
+const initialState = fromJS([]);
+
+export default (state = initialState, action) => {
```

Comment:
I'm open to changing my mind of course. I just don't understand the benefit of having a sorted array in the state (as opposed to sorting at the UI level). Could you elaborate a little on that?